spec.html

```diff
@@ -26512,7 +26533,7 @@ <h1>prototype</h1>
 
     <emu-clause id="sec-hosthassourcetextavailable">
       <h1>HostHasSourceTextAvailable ( _func_ )</h1>
-      <p>HostHasSourceTextAvailable is an implementation-defined abstract operation that allows host environments to prevent the source text from being provided for a given function.</p>
+      <p>HostHasSourceTextAvailable is an host-defined abstract operation that allows host environments to prevent the source text from being provided for a given function.</p>
```

Comment:
```suggestion
      <p>HostHasSourceTextAvailable is a host-defined abstract operation that allows host environments to prevent the source text from being provided for a given function.</p>
```